test/mock/common/filesystem/generate.go

```diff
@@ -0,0 +1,3 @@
+package filesystem_mock
+
+//go:generate mockgen -source ../../../../pkg/agent/plugin/workloadattestor/k8s/filesystem.go -destination filesystem.go -package filesystem_mock
```

Comment:
what's the motivation to have custom package name? it would be good to keep the names consistent (most of the others here have the default package name)

Reply:
I was just keeping things the same. If we want to get more aggressive, there are even more changes we can make in the name of consistency :)